src/lib/sendEmail.ts

```diff
@@ -0,0 +1,29 @@
+import sgMail from '@sendgrid/mail';
+import { emailMsg } from '../types/email.types';
+import { log } from './log';
+
+// return true if email was sent succesfully.
+export const sendEmail = async (msg: emailMsg): Promise<boolean> => {
+  const apiKey: string | undefined = process.env.SENDGRID_API_KEY;
+  const from: string | undefined = process.env.SENDGRID_FROM_EMAIL;
+
```

Comment:
Näitä ei varmaan tarvii luoda uudestaan joka kerta kun funktio ajetaan, eli nää vois ottaa ulos funktiosta ja nimetä CAPITAL_SNAKE_CASElla